scripts/cipublish

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n ${RASTER_VISION_DEBUG} ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Publish container images to Quay.io.
+"
+}
+
+docker login -u="${QUAY_USER}" -p="${QUAY_PASSWORD}" quay.io
+
+docker tag "raster-vision-${IMAGE_TYPE}" \
+    "quay.io/azavea/raster-vision:${IMAGE_TYPE}-${IMAGE_VERSION}"
+docker push "quay.io/azavea/raster-vision:${IMAGE_TYPE}-${IMAGE_VERSION}"
```

Comment:
Why the lack of our usual check for script execution vs. sourcing (e.g., `${BASH_SOURCE[0]} == "${0}"`)?

Reply:
Just an oversight here, I'll get that added in. Thanks for the catch!